utilities/ideascale-importer/ideascale_importer/tests/test_data/config.json

```diff
@@ -0,0 +1,14 @@
+{  
+    "event_id": 0,
+    "group_id": 37429,
+    "review_stage_ids": [171],
+    "nr_allocations": [1, 1],
```

Comment:
I'm having trouble understanding the meaning of `nr_allocations`. Can you clarify what `nr` refers to?

Reply:
nr - number of allocations. I will rename it